<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Roboto&quot;;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="picbox">
             <property name="text">
              <string>Load local image, or URL</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Source</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QCheckBox" name="stream">
               <property name="text">
                <string>Live Stream</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="applyURL">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Apply URL</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="urlBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="urlLabel">
                <property name="text">
                 <string>URL</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="url"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QToolButton" name="loadLocal">
               <property name="text">
                <string>Select Local Image</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="resetImage">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="parameters">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="blurLabel">
                 <property name="text">
                  <string>Blur</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="blurSlider">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="thresholdLabel">
               <property name="text">
                <string>Threshold</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="thresholdSlider">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>127</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Contrast</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="contrastSlider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>stream</sender>
   <signal>toggled(bool)</signal>
   <receiver>urlBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stream</sender>
   <signal>toggled(bool)</signal>
   <receiver>applyURL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stream</sender>
   <signal>toggled(bool)</signal>
   <receiver>loadLocal</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
